src/js/components/shepherd-element.svelte

```diff
@@ -120,16 +120,19 @@
     opacity: 0;
     outline: none;
     transition: opacity 0.3s;
```

Comment:
You probably want to add `visibility` to the transitioned properties:

```suggestion
    transition: opacity 0.3s, visibility 0.3s;
```

Otherwise it will instantly show the step and you won't see the `opacity` transition.
This feels a bit counter intuitive as `visibility` isn't a calculable property, but it works :)

https://codepen.io/snap/pen/dyYdjQe